### PR TITLE
Instagram profile avatar option added

### PR DESCRIPTION
**Pull request by orahul1**
Instagram profile avatar can add now through the Instagram user name.

### CODE REVIEW

**Comment by orahul1** [comment]
@HaithemMosbahi Please update the doc if you merge this to master.

**Comment by odahcam** [comment]
It would be nice to have the docs before merging it to master.

**Comment by orahul1** [comment]
okay.. I will add.

File: projects/ngx-avatar/src/lib/sources/instagram.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    return `https://www.instagram.com/${this.sourceId}/?__a=1`;
+  }
+
+  /**
+   * extract instagram avatar from json data
+   */
+  public processResponse(data: any, size?: number): string {
+    console.log(data)
+    if(size){
+    return `${data.graphql.user.profile_pic_url_hd}&s=${size}`;
```
Comment (odahcam) [inline projects/ngx-avatar/src/lib/sources/instagram.ts:25]: I would like to see the indentation fixed here.

File: projects/ngx-avatar/src/lib/sources/instagram.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import { AsyncSource } from './async-source';
-import { AvatarSource } from './avatar-source.enum';
+import { AsyncSource } from "./async-source";
```
Comment (odahcam) [inline projects/ngx-avatar/src/lib/sources/instagram.ts:1]:
```suggestion
import { AsyncSource } from './async-source';
```

File: projects/ngx-avatar/src/lib/sources/instagram.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
+import { AvatarSource } from "./avatar-source.enum";
```
Comment (odahcam) [inline projects/ngx-avatar/src/lib/sources/instagram.ts:2]:
```suggestion
import { AvatarSource } from './avatar-source.enum';
```

File: README.md [hunk 4]
```diff
@@ -89,6 +90,7 @@ Once the AvatarModule is imported, you can start using the component in your Ang
 <ngx-avatar facebookId="1508319875"></ngx-avatar>
 <ngx-avatar googleId="1508319875"></ngx-avatar>
 <ngx-avatar twitterId="1508319875"></ngx-avatar>
+<ngx-avatar instagramId="dccomics" size="70"></ngx-avatar>
```
Comment (odahcam) [inline README.md:93]: In fact, should've been this way:
```html
<ngx-avatar instagramId="marvel" size="70"></ngx-avatar>
```

**Pull request merged**